docs/_pages/introduction.md

```diff
@@ -170,3 +170,49 @@ Expected string to be "Expected" with a length of 8, but "Actual" has a length o
 
 For more information take a look at the [AssertionScopeSpecs.cs](https://github.com/fluentassertions/fluentassertions/blob/master/Tests/FluentAssertions.Specs/Execution/AssertionScopeSpecs.cs) in Unit Tests.
 
+### Scoped `IValueFormatter`s
```

Comment:
🔧 This section should be covered in both https://fluentassertions.com/extensibility/#rendering-objects-with-beauty and https://fluentassertions.com/introduction#assertion-scopes